src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -48,15 +46,6 @@ public static String getEditPersonDescriptorDetails(EditPersonDescriptor descrip
         descriptor.getName().ifPresent(name -> sb.append(PREFIX_NAME).append(name.fullName).append(" "));
         descriptor.getPhone().ifPresent(phone -> sb.append(PREFIX_PHONE).append(phone.value).append(" "));
         descriptor.getEmail().ifPresent(email -> sb.append(PREFIX_EMAIL).append(email.value).append(" "));
```

Comment:
Edited person will not have it's done status shown? Why is this so?

Reply:
Because done is set to false by default

Reply:
alright noted